src/ncp-spinel/SpinelNCPTaskSendCommand.cpp

```diff
@@ -210,42 +263,82 @@ nl::wpantund::SpinelNCPTaskSendCommand::vprocess_event(int event, va_list args)
 	// to execute.
 	EH_WAIT_UNTIL(EVENT_STARTING_TASK != event);
 
-	require(mNextCommand.size() < sizeof(GetInstance(this)->mOutboundBuffer), on_error);
+	if (mLockProperty != 0) {
+		mNextCommand = SpinelPackData(
+			SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_BOOL_S),
+			mLockProperty,
+			true
+		);
+
+		EH_SPAWN(&mSubPT, vprocess_send_command(event, args));
+
+		mRetVal = mNextCommandRet;
+
+		// In case of BUSY error status (meaning the `ALLOW_LOCAL_NET_DATA_CHANGE`
+		// was already true), allow the operation to proceed.
```

Comment:
Suggestion: "meaning `mLockProperty` was already true...." instead of "meaning the `ALLOW_LOCAL_NET_DATA_CHANGE`"